cypress/support/e2e.js

```diff
@@ -75,7 +75,7 @@ beforeEach(() => {
       },
       (req) => {
         req.on('response', (res) => {
-          res.setDelay(2000); // 2 seconds delay
+          res.setDelay(800); // 2 seconds delay
```

Comment:
Why 0.8 sec?
```suggestion
          res.setDelay(800); // 0.8 seconds delay
```

Reply:
Just thought that 800ms would be a suitable high latency threshold.